py/desispec/sky.py

```diff
@@ -1160,7 +1159,6 @@ def subtract_sky(frame, skymodel, throughput_correction = False, default_through
             
             if mcoeferr>np.abs(mcoef-1) :
                 log.warning("throughput corr error = %5.4f is too large compared to the correction value = %5.4f for fiber #%03d, do not apply correction"%(mcoeferr,(mcoef-1),fiber))
-                throughput_correction_value = default_throughput_correction
```

Comment:
turns out this was needed, because otherwise `throughput_correction_value` isn't set in this case and the code crashes a few lines later when it is used.  I updated directly to master to set this to 1.0.  We can chase separately what causes this warning special case and whether we should do something different about it, but for the moment I was looking for the minimal fix to get back and running.